index.html

```diff
@@ -0,0 +1,73 @@
+                    <input class="search-box" type="text" name="search" />
+                </div>
+            </div>
+        </header>
+        <section class="main-content">
+            <div class="main-text">
+                <a href=""><img class="main-text1" src="./assets/main-img1.png" alt="textimage"></a>
+                <a href=""><img src="./assets/main-img2.png" alt="textimage1"></a>
+                <a href=""><img class="main-text3" src="./assets/main-text3.png" alt="textimage2"></a>
+            </div>
+        </section>
+        <section class='bottom-grid'>
+            <img src="./assets/bottom-img.jpg" alt="bottomimage1">
+            <img src="./assets/bottom-img2.jpg" alt="bottomimage2">
+            <img src="./assets/bottom-img1.jpg" alt="bottomimage3">
+            <img src="./assets/bottom-img4.jpg" alt="bottomimage4">
+        </section>
+        <footer>
+            <section class="footer-nav">
+                <div>
+                    <p>Shop the <a>Apple Online Store</a> (1-800-MY-APPLE), visit an <a>Apple Retail Store</a>, or find
+                        a
+                        <a>reseller</a>.</p>
+                    <p>Copyright © 2014 Apple Inc. All rights reserved. <a class="footer-text1">Terms of Use</a> <a
```

Comment:
- [ ]  The `<a>` tags are missing the `href` attribute

Reply:
Changes made, Thank you